arch/arm/src/stm32h7/stm32_oneshot.c

```diff
@@ -183,14 +183,14 @@ int stm32_oneshot_initialize(struct stm32_oneshot_s *oneshot, int chan,
 {
   uint32_t frequency;
 
-  tmrinfo("chan=%d resolution=%d usec, USEC_PER_SEC:%d\n", chan, resolution,
+  tmrinfo("chan=%d resolution=%d usec, USEC_PER_SEC:%ld\n", chan, resolution,
```

Comment:
Minor. Not directly related. Optional

```suggestion
  tmrinfo("chan=%d resolution=%u usec, USEC_PER_SEC:%ld\n", chan, resolution,
```